uploadgui/src/main/scala/se/lu/nateko/cp/meta/upload/formcomponents/FileInput.scala

```diff
@@ -40,9 +41,8 @@ class FileInput(elemId: String, cb: () => Unit){
 			_hash = fail("hashsum is being computed")
 			cb()
 		}
-		if(f.size > 2000000000) {
-			UploadApp.showAlert("This file could be too large to upload with this form. " +
-				"Please contact us if you need an alternative.", "alert alert-warning")
+		if(f.size > FileMaxSize) {
+			UploadApp.showAlert(FileMaxSizeMessage, "alert alert-warning")
```

Comment:
I think we can remove this warning, we would always show the error a couple of seconds later.

Reply:
Oh right, it was much faster for me. It might depend on the file size and computer. It looks like the hasher would fail for files larger than 2GB in any case, right? Should we directly show the error and not even attempt to hash?